Client/Frontend/Fakespot/FakespotViewController.swift

```diff
@@ -107,7 +111,7 @@ class FakespotViewController: UIViewController, Themeable, UIAdaptivePresentatio
     override func viewDidDisappear(_ animated: Bool) {
         super.viewDidDisappear(animated)
         if presentingViewController == nil {
-            recordTelemetry()
+            recordDismissVCTelemetry()
```

Comment:
This should only be tracked if the user closed the bottom sheet - so not if it was dismissed because the user opted out.